src/dymaptic.GeoBlazor.Core/Components/Renderers/Renderer.cs

```diff
@@ -70,6 +90,18 @@ public enum RendererType
 {
 #pragma warning disable CS1591
     Simple,
-    UniqueValue
+    UniqueValue,
+    [LookupType("dymaptic.GeoBlazor.Pro.Components.Renderers.PieChartRenderer")]
+    PieChart
 #pragma warning restore CS1591
+}
+
+public class LookupTypeAttribute : Attribute
```

Comment:
Are we going to want a similar pattern created in the "custom" imageryrenderer class that is being setup?

Reply:
This is only necessary because of needing the full type of GB Pro classes, and the fact that the `Renderer.Type` was already implemented as an enum and not a string. Most of our `Type` properties are strings, which make this easier.

Reply:
should there be a TODO to switch this to a string for the next major release then?